multi-agent/Dockerfile.bkp

```
@@ -0,0 +1,62 @@
+# Ensure the CA bundle is readable by all users
+RUN chmod 644 /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+
+# Set environment variables to use the system bundle (which now includes your company CA + public CAs)
+ENV SSL_CERT_FILE=/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+ENV REQUESTS_CA_BUNDLE=/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+ENV CURL_CA_BUNDLE=/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+    
+# USER default
+# RUN ls -lia
+
+ADD  --chown=default:root --chmod=777 ./multi-agent $HOME/app
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)